src/test/java/org/ow2/petals/flowable/rest/RestApiTest.java

```diff
@@ -0,0 +1,151 @@
+
+    protected static final String INONLY_ENDPOINT = "edpInOnly";
+
+    protected static final ComponentUnderTest COMPONENT_UNDER_TEST = new ComponentUnderTest()
+            .addLogHandler(IN_MEMORY_LOG_HANDLER.getHandler())
+            .registerServiceToDeploy(INONLY_SU, new ServiceConfigurationFactory() {
+                @Override
+                public ServiceConfiguration create() {
+
+                    final URL wsdlUrl = Thread.currentThread().getContextClassLoader()
+                            .getResource("su/in-only/in-only.wsdl");
+                    assertNotNull("WSDL not found", wsdlUrl);
+                    final ProvidesServiceConfiguration serviceConfiguration = new ProvidesServiceConfiguration(
+                            INONLY_INTERFACE, INONLY_SERVICE, INONLY_ENDPOINT, wsdlUrl);
+
+                    final URL startResponseXslUrl = Thread.currentThread().getContextClassLoader()
+                            .getResource("su/in-only/startResponse.xsl");
+                    assertNotNull("Output XSL 'startResponse.xsl' not found", startResponseXslUrl);
+                    serviceConfiguration.addResource(startResponseXslUrl);
+
+                    final URL bpmnUrl = Thread.currentThread().getContextClassLoader()
+                            .getResource("su/in-only/in-only.bpmn");
+                    assertNotNull("BPMN file not found", bpmnUrl);
+                    serviceConfiguration.addResource(bpmnUrl);
+
+                    serviceConfiguration.setServicesSectionParameter(
+                            new QName(FlowableSEConstants.NAMESPACE_SU, "process_file"), "in-only.bpmn");
+                    serviceConfiguration
+                            .setServicesSectionParameter(new QName(FlowableSEConstants.NAMESPACE_SU, "version"), "1");
+
+                    return serviceConfiguration;
+                }
+            });
+
+    @ClassRule
+    public static final TestRule chain = RuleChain.outerRule(TEMP_FOLDER).around(IN_MEMORY_LOG_HANDLER)
+            .around(COMPONENT_UNDER_TEST);
+
+    @Override
+    protected ComponentUnderTest getComponentUnderTest() {
+        return COMPONENT_UNDER_TEST;
+    }
+
+    private WebTarget client() {
+        return client(true);
+    }
+
+    private WebTarget client(boolean auth) {
+        ClientBuilder builder = ClientBuilder.newBuilder();
+        //builder.register(new LoggingFeature(Logger.getLogger("test"), Level.SEVERE, Verbosity.PAYLOAD_TEXT, null));
+        if (auth) {
+            builder.register(HttpAuthenticationFeature.basic("kermit", "kermit"));
+        }
+        return builder.build().target("http://" + FlowableSEConstants.DEFAULT_ENGINE_REST_API_ADDRESS + ":"
+                + FlowableSEConstants.DEFAULT_ENGINE_REST_API_PORT);
+    }
+
+    @Test
+    public void getOnApiNoLogin() {
+        Response response = client(false).path("flowable-rest/service/repository/deployments")
```

Comment:
Can we replace, in URL, `flowable-rest/service/repo...` by `flowable-rest-api/repo...` ?